Libs/MRML/DisplayableManager/vtkMRMLViewDisplayableManager.cxx

```diff
@@ -394,6 +410,16 @@ void vtkMRMLViewDisplayableManager::vtkInternal::UpdateAxisVisibility()
   this->External->RequestRender();
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLViewDisplayableManager::vtkInternal::UpdateAxisColor()
+{
+  double boxColor[3];
```

Comment:
It's good practice to initialize any array with values. Above as well please.

Reply:
Fixed